### PR TITLE
usb_device: remove necessary package

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: libvirt/tests/cfg/usb_device.cfg [hunk 1]
```diff
@@ -3,7 +3,7 @@
     start_vm = "no"
     start_timeout = "60"
     usb_index = "0"
-    pkgs_host = "usbutils,usbredir-server"
```
Comment (qiankehan) [inline libvirt/tests/cfg/usb_device.cfg:6]: NACK.
The cfg file mentions the USB redirdev. The redirdev of TCP type requires usbredir-server.

Comment (dzhengfy) [reply libvirt/tests/cfg/usb_device.cfg:6]: When the pr is in draft, we do not need to review it. @qiankehan

**Comment by dzhengfy** [comment]
Tested in 9.2 with this pr:


Test name | Duration | Status
-- | -- | --
passthrough.vid_pid.usb3_0.nec_xhci_hub.pcie-root.pcie-root-port.pcie-to-pci-bridge | 1 min 39 sec | Fixed

File: libvirt/tests/cfg/usb_device.cfg (hunk 1 above)
Comment (dzhengfy) [reply libvirt/tests/cfg/usb_device.cfg:6]: > NACK. The cfg file mentions the USB redirdev. The redirdev of TCP type requires usbredir-server.

I only remove the dependency for passthrough cases. The redirdev cases  still  requires this package.@qiankehan

**Review by qiankehan — APPROVED** [review]
LGTM

**Comment by dzhengfy** [comment]
I will paste latest full test result.

**Comment by dzhengfy** [comment]
[Package](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[Duration](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[Fail](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[(diff)](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[Skip](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[(diff)](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[Pass](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[(diff)](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[Total](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)	[(diff)](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/#)
[rhel](https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Results%20Dashboard/view/RHEL-9.2-x86_64-Report/job/libvirt-RHEL-9.2-runtest-x86_64-function-usb/6/testReport/rhel/)	51 min	1	+1	1	+1	26	+25	28	+27

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**